app/views/site/about.html.erb

```diff
@@ -63,8 +63,8 @@
           <h2 class="flex-grow-1 mb-0"><%= t ".open_data_title" %></h2>
         </div>
         <p>
-          <%= t ".open_data_1_html", :reuse_link => link_to(t(".open_data_1_reuse"),
-                                                            t(".open_data_1_reuse_url")) %>
+          <%= t ".open_data_1_1_html", :reuse_link => link_to(t(".open_data_1_1_reuse"),
+                                                              t(".open_data_1_1_reuse_url")) %>
```

Comment:
We needed to rename this to `open_data_1_1_html` because some localizations already have `open_data_1_html` with different interpolation variables. Naming it `open_data_1_html` would’ve led to the same problems that #264 caused.

Reply:
(This issue should’ve been caught by #994.)